ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/components/customButton/custom-button.types.ts

```diff
@@ -0,0 +1,5 @@
+import {GestureResponderEvent} from 'react-native';
+export type CustomButtonType = {
+  title: string;
+  onPressFunction: (event: GestureResponderEvent) => void;
```

Comment:
Try to make interfaces as narrow as possible. So here instead of passing `(event: GestureResponderEvent) => void;` it would be nice to have just `() => void;`
This way you reduce coupling and support good Interface segregation principle approach.